miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/security/DefaultAuthorizationManager.java

```diff
@@ -130,9 +130,13 @@ public void throwIfNotWritable(SecurableByProfile resource) throws IOException,
     return filtered;
   }
 
+  /**
+   * Does not throw if owner is null (editable by anyone).
+   */
   @Override
   public void throwIfNonAdminOrMatchingOwner(User owner) throws IOException, AuthorizationException {
     User currentUser = getCurrentUser();
+    if (owner == null) return;
```

Comment:
I think that if the owner doesn't exist because they're no longer there, it should be editable by other lab members (who aren't admins) because dilutions should be editable by all lab members in the same way that samples and libraries are. Perhaps it could be restricted to editing if the currentUser has permissions on the dilution, but isn't necessarily the owner?